plugins/ps/functions

```diff
@@ -21,17 +21,21 @@ generate_scale_file() {
     trap 'rm -rf "$TMP_WORK_DIR" > /dev/null' RETURN
 
     copy_from_image "$IMAGE" "/app/Procfile" "$TMP_WORK_DIR" 2>/dev/null || true
+    copy_from_image "$IMAGE" "/opt/Procfile.compose" "$TMP_WORK_DIR" 2>/dev/null || true
```

Comment:
I needed to store this file somewhere during the `dokku build` step, and `/app/` doesn't exist there yet...